src/hunspell/hunspell_wrapper.h

```diff
@@ -0,0 +1,27 @@
+#ifndef __HUNSPELL_WRAPPER_H__
+#define __HUNSPELL_WRAPPER_H__
```

Comment:
Double underscores are reserved, it's best to avoid them
```suggestion
#ifndef HUNSPELL_WRAPPER_H
#define HUNSPELL_WRAPPER_H
```